### PR TITLE
Upgrades to latest fastlz version

### PR DESCRIPTION
**Pull request by randall**
Move to lib/fastlz to remind people it exists
    
Removed reference to lib/wcpp and lib/tsconfig from NOTICE. These were
dropped in ATS 9.x

### CODE REVIEW

**Comment by ywkaras** [comment]
+1 your intentions.  But when I've tried to do cleanup PRs, no matter how big the mess it's cleaning up, people seem to feel that, if it seems to be working, don't even breath on it.

File: src/tscore/fastlz.c [hunk 1]
```diff
@@ -24,512 +21,615 @@
...
+    memmove(dest, src, count);
+  } else {
+    switch (count) {
+    default:
+      do {
+        *dest++ = *src++;
+      } while (--count);
+      break;
+    case 3:
+      *dest++ = *src++;
```
Comment (ywkaras) [inline src/tscore/fastlz.c:98]: https://en.cppreference.com/w/cpp/language/attributes/fallthrough

Comment (ywkaras) [reply src/tscore/fastlz.c:98]: Oh wait it's C.  Maybe just change the extension?  Or else:  https://stackoverflow.com/questions/45349079/how-to-use-attribute-fallthrough-correctly-in-gcc

File: src/tscore/fastlz.cc [hunk 2]
```diff
@@ -0,0 +1,635 @@
...
+    memmove(dest, src, count);
+  } else {
+    switch (count) {
+    default:
+      do {
+        *dest++ = *src++;
+      } while (--count);
+      break;
+    case 3:
+      *dest++ = *src++;
```
Comment (ywkaras) [inline src/tscore/fastlz.cc:98]: You still need to add `[[fallthrough]]` here and after lines 100 and 102.

Comment (masaori335) [reply src/tscore/fastlz.cc:98]: `[[fallthrough]]` is C23. I think we need to move this to `/lib/` and compile with cc.
https://en.cppreference.com/w/c/language/attributes/fallthrough

Comment (randall) [reply src/tscore/fastlz.cc]: `[[fallthrough]]` added. Moved to lib/fastlz and renamed to .cc

**Comment by randall** [comment]
[approve ci autest]

**Review by ywkaras — APPROVED** [review]

**Comment by randall** [comment]
I almost gave up here. No good deed goes unpunished.

**Pull request merged**